content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -158,6 +158,12 @@ To use the `systemd` cgroup driver in `/etc/containerd/config.toml` with `runc`,
     SystemdCgroup = true
 ```
 
+If you apply this change make sure to restart containerd again:
+
+```shell
+sudo systemctl restart containerd
```

Comment:
Mostly, we need `systemctl daemon-reload`.
Not sure that it is needed here.

Reply:
After I checked other pages with `systemctl restart containerd`,  `systemctl daemon-reload` is not mentioned.

Reply:
yeah, deamon-reload is not needed when reloading the containerd config.yaml.
only when the service files are changed.